projects.md

```diff
@@ -0,0 +1,6 @@
+TODO: Make this a comprehensive guide about how to find the desired project and provide an overview about all available projects/repositories
+
+// TODO: provide a list of all SimRel (or even all Eclipse projects), ideally something that's managed by the EF, but it should also help  // interested contributors to find the corresponding project.
+// TODO: tips to identify the Plug-in of interest?
+// - Alt+Shift+F1
+// - Repository references encoded in Plug-in MANIFEST.MF files?!
```

Comment:
My idea was to add the overview of Eclipse IDE projects here.
In general what you have started in the [page-projects](https://github.com/eclipse-ide/.github/tree/page-projects) branch.